diff --git a/rootdir/init.environ.rc.in b/rootdir/init.environ.rc.in
index 67222d7..6f37ffd 100644
--- a/rootdir/init.environ.rc.in
+++ b/rootdir/init.environ.rc.in
@@ -10,4 +10,4 @@ on init
     export LOOP_MOUNTPOINT /mnt/obb
     export BOOTCLASSPATH %BOOTCLASSPATH%
     export SYSTEMSERVERCLASSPATH %SYSTEMSERVERCLASSPATH%
-    export LD_PRELOAD libsigchain.so
+    export LD_PRELOAD libfd_tracker.so:libsigchain.so
